homeassistant/components/velux/entity.py

```diff
@@ -19,7 +23,24 @@ def __init__(self, node: Node, config_entry_id: str) -> None:
             if node.serial_number
             else f"{config_entry_id}_{node.node_id}"
         )
-        self._attr_name = node.name if node.name else f"#{node.node_id}"
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return device info."""
+        return DeviceInfo(
```

Comment:
Instead set this to `_attr_device_info` in the constructor, and you can assign DeviceInfo fields directly
```py
DeviceInfo(
    identifiers={(...
```